projects/assetforfeiture/index.md

```diff
@@ -17,6 +17,6 @@ You can read our full investigation, "Inside the Chicago Police Department's Sec
 
 Following our investigation, there were calls for reform in the [Chicago City Council](https://www.chicagoreader.com/Bleader/archives/2016/10/03/city-hall-reacts-to-reader-revelations-about-the-chicago-police-departments-secret-budget) and the Illinois legislature. While city leaders expressed concern that the police department was able to use forfeiture money at its own discretion, and had hidden the funds from the City Council and the mayor's office, there have so far been no proposals to change the practice within the city government.
 
-However, in September, Illinois adopted a new law titled the Seizure and Forfeiture Reporting Act, which will bring some reforms to the process beginning in January 2018. (In a letter supporting the reform bill, Cook County State's Attorney Kim Foxx cited [our research](https://lucyparsonslabs.com/journalism/CivilAssetForfeitureReform/).)  The most important changes will be that CPD and other law enforcement agencies across the state will need to report publicly the money they make and spend through civil forfeiture; that small amounts of marijuana can no longer justify the seizure and forfeiture of cash or other assets, less than $500; and that the burden of proof will be on the state, as opposed to the victim. (Currently, the state only needs to prove that police and prosecutors have probable cause that the asset was connected to a crime, while a person seeking to get their property back needs to meet the higher standard of a "preponderance of evidence" to prove they were in possession of the property legally.)
+However, in September, Illinois adopted a new law titled the Seizure and Forfeiture Reporting Act, which will bring some reforms to the process beginning in January 2018. (In a letter supporting the reform bill, Cook County State's Attorney Kim Foxx cited [our research](https://lucyparsonslabs.com/posts/HitsThePoor/).)  The most important changes will be that CPD and other law enforcement agencies across the state will need to report publicly the money they make and spend through civil forfeiture; that small amounts of marijuana can no longer justify the seizure and forfeiture of cash or other assets, less than $500; and that the burden of proof will be on the state, as opposed to the victim. (Currently, the state only needs to prove that police and prosecutors have probable cause that the asset was connected to a crime, while a person seeking to get their property back needs to meet the higher standard of a "preponderance of evidence" to prove they were in possession of the property legally.)
```

Comment:
Nit but I believe the line should be `(In a letter supporting the reform bill, Cook County State's Attorney Kim Foxx cited [our research.](https://lucyparsonslabs.com/posts/HitsThePoor/))` This should be AP-style compliant.